utils/utils.go

```diff
@@ -687,3 +693,32 @@ func SendError(w http.ResponseWriter, statusCode int, message string) {
 	w.WriteHeader(statusCode)
 	json.NewEncoder(w).Encode(map[string]string{"error": message})
 }
+
+func updateUploadStatus(db *sql.DB, videoID string) error {
```

Comment:
here a if your `updateUploadStatus` if you see the function name is very generic but what you are doing inside is very specific. So I suggest you also take the status as a parameter in the function and pass it on wherever you are updating the status.

this function will become re-usable.